misc/toml-parser/src/main/java/io/ballerina/toml/semantic/ast/TomlTransformer.java

```diff
@@ -66,14 +67,48 @@ public TomlTransformer() {
         this.dlog = DiagnosticLog.getInstance();
     }
 
+    /**
+     * This method gets called at the start of the transforming phase as DocumentNode is the root node of the ST.
+     *
+     * @param documentNode root node of the toml syntax tree
+     * @return transformed root toml table
+     */
     @Override
     public TomlNode transform(DocumentNode documentNode) {
         TomlTableNode rootTable = createRootTable(documentNode);
 
         NodeList<DocumentMemberDeclarationNode> members = documentNode.members();
         for (DocumentMemberDeclarationNode rootNode : members) {
+            //DocumentMemberDeclarationNode contains root level nodes in the document. This could contain
```

Comment:
When writing comments as a practice we keep a space like this
```suggestion
            // DocumentMemberDeclarationNode contains root level nodes in the document. This could contain
```